test/pub_uploader_test.dart

```diff
@@ -31,7 +31,7 @@ See http://dartlang.org/tools/pub/cmd/pub-uploader.html for detailed documentati
 ScheduledProcess startPubUploader(ScheduledServer server, List<String> args) {
   var tokenEndpoint = server.url.then((url) =>
       url.resolve('/token').toString());
-  args = ['uploader', '--server', tokenEndpoint]..addAll(args);
+  args = <Object>['uploader', '--server', tokenEndpoint]..addAll(args);
```

Comment:
Why does this need to be annotated as `Object`? Isn't that the LUB for `Future<String>` and `String`?